### ISSUE TITLE
Port Issue: VCMI update / video playback

### ISSUE THREAD

**Issue by Laserlicht** [PortsMaster/PortMaster-New#497]
#### Description
- Update to VCMI 1.5.3 (use master branch for this)
- Video playback won't work (at least on Anbernic rg35xx h)

**Comment by ddrsoul** [comment]
updated to 15.08.24 dev branch version with gamepad support

**Comment by JeodC** [comment]
> updated to 15.08.24 dev branch version with gamepad support

Does that mean this issue should be closed, or?

**Comment by ddrsoul** [comment]
> > updated to 15.08.24 dev branch version with gamepad support
> 
> Does that mean this issue should be closed, or?

not really, video is still broken and I have troubles compiling vcmi with ffmpeg on ubuntu 20.04. maybe it will work on more recent ubuntu but then we will abandon arkos? or maybe my skills are just not good enough to solve hundreds of linking errors caused by ffmpeg on 20.04

**Comment by ddrsoul** [comment]
After compiling vcmi 1.6.5 with ffmpeg inside debian bookworm I still didn't manage to run it with videoclips. I give up for now.

**Comment by JeodC** [comment]
> After compiling vcmi 1.6.5 with ffmpeg inside debian bookworm I still didn't manage to run it with videoclips. I give up for now.

According to https://github.com/vcmi/vcmi/pull/5438 video should be working now. Can you verify?

**Comment by JeodC** [comment]
> > > After compiling vcmi 1.6.5 with ffmpeg inside debian bookworm I still didn't manage to run it with videoclips. I give up for now.
> > 
> > 
> > According to [vcmi/vcmi#5438](https://github.com/vcmi/vcmi/pull/5438) video should be working now. Can you verify?
> 
> Yes. But this version is currently only on discord: https://discord.com/channels/1122861252088172575/1333447929066819635/1344753362524176526

That still means it can be compiled and thus fixed. @ddrsoul

**Comment by ddrsoul** [comment]
New build made by Laserlicht is packed into autoinstall zip and posted in discord test thread

https://discordapp.com/channels/1122861252088172575/1278344771869478982

video playback fixed, version bumped to lastest stable (1.6.7)
not sure about arkos status, need some testers
but I think we should just exclude arkos support if it is broken

**Comment by JeodC** [comment]
> New build made by Laserlicht is packed into autoinstall zip and posted in discord test thread
> 
> https://discordapp.com/channels/1122861252088172575/1278344771869478982
> 
> video playback fixed, version bumped to lastest stable (1.6.7) not sure about arkos status, need some testers but I think we should just exclude arkos support if it is broken

Status of this? Was there a pull request not linked to the issue?

**Comment by ddrsoul** [comment]
> > New build made by Laserlicht is packed into autoinstall zip and posted in discord test thread
> > https://discordapp.com/channels/1122861252088172575/1278344771869478982
> > video playback fixed, version bumped to lastest stable (1.6.7) not sure about arkos status, need some testers but I think we should just exclude arkos support if it is broken
> 
> Status of this? Was there a pull request not linked to the issue?

I did not manage to fix the bug with sway on rocknix. with small community demand vcmi update has stalled
this is the reference message (i hope it is ok to attach discord links)

https://discord.com/channels/1122861252088172575/1122882437291188345/1352533105935650857
